packages/generator/src/helper.ts

```diff
@@ -82,7 +82,10 @@ export const preprocessing = async (arg: { template: Template; userPlugins: Plug
       : Object.values(builtInPlugins)
   ) as Plugin<Schema>[];
 
-  const schemaTypes = schemas.flatMap(schemaPage => schemaPage.map((schema) => schema.type));
+  let schemaTypes = schemas.flatMap(schemaPage => schemaPage.map((schema) => schema.type));
+  if (schemaTypes.indexOf('text') == -1) {
```

Comment:
I totally understand what you are trying to do. thanks.

but i will create another pr, cuz I come up with more robust way.